tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -314,14 +314,12 @@ def _add_expectations_and_checkpoint(
         checkpoint_config["runtime_configuration"] = dict_to_update_checkpoint
 
     context: DataContext = data_context
-    context.add_expectation_suite(expectation_suite_name="metrics_exp")
+    context.add_or_update_expectation_suite(expectation_suite_name="metrics_exp")
```

Comment:
So that we don't get warnings anymore :)

Reply:
woohooo ty

Reply:
why is `add_expectation_suite` giving warnings? its a new method (compared to `create_expectation_suite`)

Reply:
Ah I think you're right :) Thanks for the catch